### PR TITLE
new k8s versions for etcd options

### PR DESCRIPTION
**Pull request by kinarashah**
- add v1.14.9-rancher1, v1.15.6-rancher1, v1.16.3-rancher1
- service options for etcd

### CODE REVIEW

File: rke/k8s_service_options.go [hunk 1]
```diff
@@ -192,3 +200,10 @@ func getSchedulerOptions() map[string]string {
 		"profiling":    "false",
 	}
 }
+
+func getETCDOptions() map[string]string {
```
Comment (leodotcloud) [inline rke/k8s_service_options.go]: Shouldn't we add these etcd options to the other k8s versions > 1.13?

Comment (kinarashah) [reply rke/k8s_service_options.go]: could add, i thought cis scan ran only for 1.13

**Comment by alena1108** [comment]
@kinarashah k8s recently added new patch versions. I'm going to build hyperkube images and let you know the tags

**Comment by alena1108** [comment]
v1.14.9-rancher1
v1.15.6-rancher1
v1.16.3-rancher1

**Pull request merged**